dataclass.py

```diff
@@ -3,14 +3,13 @@
 #  what exception to raise when non-default follows default? currently
 #  ValueError
 
-# special name in __init__: _self: reserve this name?
```

Comment:
This was an unrelated change that I should not have pushed with this. But it's not a big deal: leaving it is easier than taking it out.